src/Tickets/Commerce/Admin_Tables/Orders.php

```diff
@@ -505,4 +505,33 @@ public function search_box( $text, $input_id ) {
 
 		echo $search_box;
 	}
+
+	/**
+	 * Displays extra controls.
+	 *
+	 * @param string $which The location of the actions: 'left' or 'right'.
```

Comment:
We need a `@since` param here:
```suggestion
	 * Displays extra controls.
	 *
	 * @since TBD
	 *
	 * @param string $which The location of the actions: 'left' or 'right'.
```

Reply:
Thanks @codingmusician! Suggestion applied.